dapr-spring/dapr-spring-boot-autoconfigure/src/main/java/io/dapr/spring/boot/autoconfigure/client/DaprClientProperties.java

```diff
@@ -0,0 +1,7 @@
+package io.dapr.spring.boot.autoconfigure.client;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
```

Comment:
Just a small question, should we use the old Java style Java Beans or should we recommend users to go with `Java records`?

Reply:
both should be fine right?

Reply:
IMO, I would go with java beans in order to add javadoc and could generate the metadata, the configuration processor is already in the classpath. See https://docs.spring.io/spring-boot/specification/configuration-metadata/annotation-processor.html